### ISSUE TITLE
Freestyler DMX 

### ISSUE THREAD

**Issue by Hirte666** [bitfocus/companion-module-requests#995]
Hello
I would love it if there would be a remote from Freestyler DMX

Freestyler is a nearly opensource DMX software to control light. DMX is the official light control protocol, GrandMA e.g. is already embedded and it outputs DMX or Artnet at the end. 
But for the very small money there is the Freestyler that works with almost any DMX interface. 
I work with it for years for front light and / or party light.

For freestyler there is an IOS/google app via network/wifi and you can control it via midi. 

I tried it with Midi Relay on the same computer ... but that was unsuccessful so far.

Website : [https://www.freestylerdmx.be/](https://www.freestylerdmx.be/)

At the beginning I would be happy the override buttons and start/stop cuelist   

Is anyone interested, I can help with the operation of the Freestyler, but with programming I have no idea.  

Translated with www.DeepL.com/Translator (free version)

**Comment by Hirte666** [comment]
Ok, theres a Tcp  protocol for Freestyler.  and ist works with   Companion Tcp/udp ;)

**Comment by josephdadams** [comment]
@Hirte666 are you content with using the generic TCP/UDP module? Or would you still like a custom module developed?

**Comment by jlinders13** [comment]
Yes, a custom module to be developed would be awesome. I use FreeStyler and need this. 
Thanks

**Comment by Hirte666** [comment]
@josephdadams   add the   Genric TCP/UDP  under Connections  ,   connect them to Freestyler  :    Target IP :  127.0.0.1 ( is FS running on Local )   Port is  3333    Connect with TCP

Buttons :   Send Command   : FSOC070255   -> for Pressing OverrideButton 1     

A full Command list should be in your FS folder :  ... \FeeStyler\Documentation\Sendmessage and TCPIP.pdf

**Comment by josephdadams** [comment]
@Hirte666

**Comment by Hirte666** [comment]
> @Hirte666

sorry , send a message a bit to fast :)

**Comment by josephdadams** [comment]
Thought we were just tagging each other 😂
